deploy-image/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+
+RUN apt-get update && apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
```

Comment:
Is there no way to line continuations to make this look a bit better?

Reply:
Yep `\\` works in RUN commands